frontend/packages/dev-console/src/components/pipelines/pipeline-overview/PipelineOverview.tsx

```diff
@@ -41,7 +40,7 @@ const PipelinesOverview: React.FC<PipelinesOverviewProps> = ({
           </Link>
         )}
       </SidebarSectionHeading>
-      <ListGroup componentClass="ul">
+      <ul className="list-group">
```

Comment:
What defines the `list-group` class?

Reply:
Bootstrap 3.  https://github.com/openshift/console/blob/master/frontend/public/vendor.scss#L23